Example.md

```diff
@@ -0,0 +1,30 @@
+# Example to test the implementation
+
+## Level 2
+
+### Level 3
+
+This is some text with *emphasis*, **double emphasis** and ***triple emphasis***.
+
+There is also a list:
+
+ * item a
+ * item b
+ * item c
+
+And an enumeration:
+
+ 1. item 1
+ 1. item 2
+ 1. item 3
+
+> Quotes should also be supported.
+
+What should we do with [links](https://github.com/dbrgn/mdr) and images ![image](https://upload.wikimedia.org/wikipedia/commons/thumb/4/48/Markdown-mark.svg/312px-Markdown-mark.svg.png "Markdown logo")?
+
+Code blocks should get fancy syntax highlighting:
+```Rust
```

Comment:
This is not valid markdown though. Only the Github flavored style.

We should still support it, as long as the Hoedown library can parse it.